client/components/home.jsx

```diff
@@ -43,28 +43,28 @@ const styles = {
 };
 
 class HomeWrapper extends React.Component {
-  constructor() {
-    super();
+  // constructor() {
+  //   super();
 
-    fetch("/getBeerStyles", {
-      credentials: "same-origin",
-      method: "GET",
-      headers: {
-        "Accept": "application/json",
-        "Content-Type": "application/json"
-      }
-    })
-    .then((resp) => {
-      if (resp.status === 200) {
-        this.setState({testResult: `GET SUCCEEDED with status ${resp.status}` });
-      } else {
-        this.setState({testResult: `GET FAILED with status ${resp.status}` });
-      }
-    })
-    .catch((e) => {
-      this.setState({testResult: `GET FAILED: ${e.toString()}`});
-    });
-  }
+  //   fetch("/getBeerStyles", {
```

Comment:
@donsalari I commented this out until we have the redux store going. Fetching on the server causes the component to not be server side rendered.